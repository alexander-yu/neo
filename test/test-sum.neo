void main() {
    auto im = [[1, 2], [3, 4]];
    auto fm = [[1., 2.], [3., 4.]];
    println(isum(im, 0));
    println(isum(im, 1));
    println(fsum(fm, 0));
    println(fsum(fm, 1));
}