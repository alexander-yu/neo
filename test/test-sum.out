[[4, 6]]
[[3],
 [7]]
[[4, 6]]
[[3],
 [7]]
